xbmc/platform/darwin/osx/CocoaInterface.mm

```diff
@@ -36,12 +32,7 @@
 
 NSOpenGLContext* Cocoa_GL_GetCurrentContext(void)
 {
-#if defined(HAS_SDL)
-  CWinSystemOSX *winSystem = dynamic_cast<CWinSystemOSX*>(CServiceBroker::GetWinSystem());
-  return winSystem->GetNSOpenGLContext();
-#else
   return [NSOpenGLContext currentContext];
-#endif
 }
 
```

Comment:
While you are in here, the following can be removed (header and implementation)

https://github.com/xbmc/xbmc/blob/d6b9189f5ca69886c4195e7695632197e27dd9f6/xbmc/platform/darwin/osx/CocoaInterface.h#L33-L36
https://github.com/xbmc/xbmc/blob/d6b9189f5ca69886c4195e7695632197e27dd9f6/xbmc/platform/darwin/osx/CocoaInterface.mm#L232-L240

Only current use is gated by HAS_SDL which you remove from xbmc/Application.cpp

Reply:
thanks for noticing, I plan to review those unused methods. Did you take a deep look or just noticed this one?
I believe a bunch of others might also be unused if SDL is dropped.

Reply:
Just noticed while dealing with some warnings for SDK bump